.github/workflows/audit.yml

```
@@ -8,7 +8,7 @@ jobs:
   audit:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2.3.4
```

wow that's a very explicit pin :|

is it possible to make it not care about minors? cc @lfrancke

---

i guess we can make a suggestion and maybe it's okay with it
```suggestion
      - uses: actions/checkout@v2
```

---

ah great. suggestions doesn't pass DCO.

---

This is the "recommended" way of doing things: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsuses

That said: These actions don't update very often.
If you think they are too annoying we can disable it entirely again.

To answer your original question: I'm afraid not, no.

---

Hehe ok. I'm okay with the threat potential of someone compromising a core github action (which i think is super unlikely), so i don't want a weekly pr to bump it. But getting the PR notifying me of a major here is useful.

Also, for third-party actions, it's probably beneficial to pin them.

---

Yeah, it's very unlikely...